develop/plone/searching_and_indexing/indexing.rst

```diff
@@ -281,64 +281,53 @@ The difference with metadata is that
 
 * You store always a value copy as is
 
-To create metadata colums in your ``catalog.xml`` add::
+To create metadata colums in your ``catalog.xml`` add:
 
-	<?xml version="1.0"?>
-	<object name="portal_catalog" meta_type="Plone Catalog Tool">
-
-		<!-- Add a new metadata column which will read from context.getSignificant() function -->
-		<column value="getSignificant"/>
+.. code-block:: xml
 
-	</object>
+    <?xml version="1.0"?>
+    <object name="portal_catalog" meta_type="Plone Catalog Tool">
+      <!-- Add a new metadata column which will read from context.getSignificant() function -->
+      <column value="getSignificant"/>
+    </object>
 
 
 When indexing happens and how to reindex manually
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Content item reindexing is run when
-
-Plone calls reindexObject() if
+Content indexing happens automatically if:
 
 * The object is modified by the user using the standard edit forms
 
 * portal_catalog rebuild is run (from *Advanced* tab)
 
-* If you add a new
-  index you need to run :doc:`Rebuild catalog </develop/plone/searching_and_indexing/catalog>`
-  to get the existing values from content objects to new index.
-
-* You might also want to call :doc:`reindexObject()
```

Comment:
I'm uncomfortable deleting material; maybe you could move what you consider to be edge case documentation to a catch-all section at the end of this page?

I also think it's preferable to summarize the forum thread information and put that summarized info here, rather than making someone read the entire thread and have to digest it to get the final info. There is also the chance that the forum thread might not be there in future (broken links).